### PR TITLE
OPENEUROPA-1508 Create News Content Type

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: runner.yml.dist [hunk 1]
```diff
@@ -23,8 +23,10 @@ drupal:
     - "./vendor/bin/drush config-set system.theme admin seven -y"
     - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
     # Enable the modules.
+    - "./vendor/bin/drush en oe_media -y"
```
Comment (upchuk) [inline runner.yml.dist]: Since oe_content_news depends on this, we don't need to enable manually no?

Comment (nagyad) [reply runner.yml.dist]: In this case, yes we don't need.

File: modules/oe_content_news/config/install/field.storage.node.oe_news_location.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - rdf_skos
+id: node.oe_news_location
+field_name: oe_news_location
+entity_type: node
+type: skos_concept_entity_reference
+settings:
+  target_type: skos_concept
+module: rdf_skos
+locked: false
+cardinality: -1
```
Comment (brummbar) [inline modules/oe_content_news/config/install/field.storage.node.oe_news_location.yml]: The cardinality of this should be 1.

File: modules/oe_content_news/oe_content_news.info.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+core: 8.x
+dependencies:
+  - oe_content:oe_content
+  - oe_media:oe_media
+  - oe_media_avportal:oe_media_avportal
+
+config_devel:
+  - core.entity_form_display.node.oe_news.default
+  - core.entity_view_display.node.oe_news.default
+  - core.entity_view_display.node.oe_news.teaser
```
Comment (brummbar) [inline modules/oe_content_news/oe_content_news.info.yml]: I missed it in the page ticket, but let's remove the teaser display as we don't have the specifications for it nor any use case at the moment.

**Comment by brummbar** [comment]
We remove the teaser from page in another PR.

**Pull request merged**